eng/build-utils.ps1

```diff
@@ -249,9 +249,7 @@ function Make-BootstrapBuild() {
     $projectPath = "$RepoRoot\proto.proj"
     Run-MSBuild $projectPath "/restore /t:Build" -logFileName "Bootstrap" -configuration $bootstrapConfiguration
     Copy-Item "$ArtifactsDir\bin\fsc\$bootstrapConfiguration\$bootstrapTfm\*" -Destination $dir
-
-    Write-Host "Cleaning Bootstrap compiler artifacts"
-    Run-MSBuild $projectPath "/t:Clean" -logFileName "BootstrapClean" -configuration $bootstrapConfiguration
```

Comment:
Why is the clean step getting removed?

Reply:
Because it just deletes a few files from fsc\Proto\net46, and looks odd.